src/components/CreateSecret/Form/Annotations.js

```diff
@@ -0,0 +1,121 @@
+/*
+Copyright 2019-2020 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from 'react';
+import { injectIntl } from 'react-intl';
+import { Button, Dropdown, TextInput } from 'carbon-components-react';
+import Remove from '@carbon/icons-react/lib/close--outline/20';
+import Add from '@carbon/icons-react/lib/add--alt/24';
+
+const itemToString = item => (item ? item.text : '');
+
+const Annotations = props => {
+  const {
+    annotations,
+    handleAdd,
+    handleAnnotationChange,
+    handleRemove,
+    intl,
+    invalidFields,
+    loading
+  } = props;
+
+  return (
+    <>
+      {annotations.map((annotation, index) => {
+        return (
+          <div className="annotationDiv" key={annotation.id}>
+            <Dropdown
+              id="accessTo"
+              titleText={intl.formatMessage({
+                id: 'dashboard.createSecret.accessTo',
+                defaultMessage: 'Access To'
+              })}
+              label=""
+              initialSelectedItem={{
+                id: annotation.access,
+                text:
+                  annotation.access === 'git' ? 'Git Server' : 'Docker Registry'
```

Comment:
These values will need to match the ones from `items` below. If using a non-English locale this may not be the case, so we'll need to update the code here a little.

Let's pull the intl values out before the return and reference them both here and in `items` so they'll always be consistent.